src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -137,6 +137,9 @@
                     <span [ngStyle]="check.severity | healthColor"
                           [class.health-warn-description]="check.severity === 'HEALTH_WARN'">
                     {{ check.type }}</span>: {{ check.summary.message }}
+                    <div *ngFor="let details of check?.detail">
+                      <div class="text-muted">{{ details?.message }}</div>
```

Comment:
can we use here `cd-help-text` ?
https://github.com/ceph/ceph/blob/a8d01fff0050c90c634d1f645d10698a4d92ca47/src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html#L185-L188